README.md

```diff
@@ -0,0 +1,49 @@
+# Hail Batch Workflows
+
+This repository provides implementations of genomics workflows using Hail Batch and the [cpg_utils/workflows library](https://github.com/populationgenomics/cpg-utils/blob/main/cpg_utils/workflows/README.md), specifically:
+
+* WES/WGS Seqr Loader: FASTQ -> CRAM -> GVCF -> pVCF -> Matrix Table -> Elasticsearch index, with an ability to use parts of this pipeline as e.g. a single-sample germline variant calling workflow (FASTQ -> GVCF), joint-calling pipeline (GVCF -> pVCF), AS-VQSR pipeline, etc.
+* GATK-SV: CRAM -> VCF and more, orchestrates [GATK-SV WDL workflows](https://github.com/broadinstitute/gatk-sv) in Hail Batch.
+
+## Seqr Loader
+
+![uml](docs/seqr_loader.png)
```

Comment:
From the diagram it looks like all the Hail Query stages are run through Dataproc. Is that correct? The last stage (writing to Elasticsearch) probably has a Spark dependency, but couldn't we run everything else in Query on Batch?

Do the pedigree checks actually happen twice?

Do you also want to add an editable version of this diagram into this repository?

Reply:
> From the diagram it looks like all the Hail Query stages are run through Dataproc. Is that correct? The last stage (writing to Elasticsearch) probably has a Spark dependency, but couldn't we run everything else in Query on Batch?

We have that query code ready to run on Batch backend, we tested that, and they can be quickly swapped to use Batch. But it was tricky to debug issues with Batch backend, so I switched to a more stable Spark backend for this production code. We can switch back in the future.

Reply:
> Do you also want to add an editable version of this diagram into this repository?

Added a Miro board URL

Reply:
Thank you! Just the one remaining question then, about the pedigree checks being shown twice in the diagram.